httputils/istio/stophook.go

```diff
@@ -0,0 +1,16 @@
+package istio
+
+import (
+	"net/http"
+	"time"
+)
+
+// StopHook takes a timeout in milliseconds and sends a request to istio's stop hook.
+// It is intended to be used as a defer function
+func StopHook(timeout int) {
+	c := &http.Client{
+		Timeout: time.Duration(timeout) * time.Milisecond,
```

Comment:
this is causing the build to fail
```suggestion
		Timeout: time.Duration(timeout) * time.Millisecond,
```